dist.html

```diff
@@ -94,8 +94,8 @@ <h3 id="measure-description" style="font-size: 14px; margin: 0 3px"></h3>
                                     <label><input name="cumulative-toggle" type="radio" value="1"> Cumulative</label>
                                 </div>
                                 <div>
-                                    Date range variable: filter submissions by build ID range or submission date range:
-                                    <label><input name="build-time-toggle" type="radio" value="0"> Build Date</label>
+                                    Date range variable: filter submissions by build/release date range or submission date range:
```

Comment:
For viewing data under the release channel, it might make more sense to say "release date" rather than "build date".